os/tools/mkbinheader.py

```diff
@@ -164,6 +164,12 @@ def get_static_ram_size(bin_type):
 COMP_MINIZ = 2
 COMP_MAX = COMP_MINIZ
 
+SCHED_PRIORITY_MIN = 1
+SCHED_PRIORITY_MAX = 255
+
+BM_PRIORITY_MIN = 200
```

Comment:
In this case, we'd better make config for binary manager.
If not, we should modify both this and binary_manager.h.
That can be some mistakes sometime.

Reply:
I agree with your opinion. but these values are changed by user. I will think about it more.

Reply:
Yes, these can be changed by user, so that can make problem.